src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/CCDConfigConstant.java

```diff
@@ -5,15 +5,29 @@
     //Case related
     public static final String STATE = "state";
     public static final String DIVORCE_CASE_NUMBER = "divorceCaseNumber";
+
+    public static final String APP_FIRST_AND_MIDDLE_NAME_CCD_FIELD = "applicantFMName";
+    public static final String APP_LAST_NAME_CCD_FIELD = "applicantLName";
+    public static final String APP_ADDRESS_CCD_FIELD = "applicantAddress";
+
+    public static final String APP_RESP_FIRST_AND_MIDDLE_NAME_CCD_FIELD = "appRespondentFMName";
+    public static final String APP_RESP_LAST_NAME_CCD_FIELD = "appRespondentLName";
+    public static final String RESP_ADDRESS_CCD_FIELD = "respondentAddress";
+
     //Solicitor related
     public static final String SOLICITOR_REFERENCE = "solicitorReference";
     public static final String SOLICITOR_NAME = "solicitorName";
     public static final String SOLICITOR_EMAIL = "solicitorEmail";
     public static final String CONTESTED_SOLICITOR_EMAIL = "applicantSolicitorEmail";
     public static final String CONTESTED_SOLICITOR_NAME = "applicantSolicitorName";
     public static final String SOLICITOR_FIRM = "solicitorFirm";
+    public static final String APP_SOLICITOR_ADDRESS_CCD_FIELD = "solicitorAddress";
+    public static final String RESP_SOLICITOR_ADDRESS_CCD_FIELD = "rSolicitorAddress";
+    public static final String APPLICANT_REPRESENTED = "applicantRepresented";
```

Comment:
hehe, I have just added it. 

No worries, I will resolve conflicts

Reply:
Can we name this "respondentSolicitorAddress" or "respSolicitorAddress"?

Reply:
if it's not too hard, that is.